students/uss_tomasz/lesson_14_decorators/14_1/sort_decorator.py

```diff
@@ -0,0 +1,14 @@
+def sort_decorator(func):
+    def inner_function():
```

Comment:
It's a minor and it was not specified within the task, but such decorators should rather pass through `*args` and `**kwargs` to decorated function similarly to other scripts in this lesson..